contract-ui/tabs/listings/components/listing-stats.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  SimpleGrid,
+  Skeleton,
+  Stack,
+  Stat,
+  StatLabel,
+  StatNumber,
+} from "@chakra-ui/react";
+import {
+  useDirectListingsCount,
+  useEnglishAuctionsCount,
+  useListingsCount,
+} from "@thirdweb-dev/react";
+import { Marketplace } from "@thirdweb-dev/sdk";
+import { MarketplaceV3 } from "@thirdweb-dev/sdk/evm";
+import { BigNumber } from "ethers";
+import { useMemo } from "react";
+import { Card } from "tw-components";
+
+interface ListingStatsProps {
+  contract?: Marketplace;
+}
+
+export const ListingStats: React.FC<ListingStatsProps> = ({ contract }) => {
+  const listingsQuery = useListingsCount(contract);
+
+  return (
+    <SimpleGrid columns={3} spacing={{ base: 3, md: 6 }}>
+      <Card as={Stat}>
+        <StatLabel mb={{ base: 1, md: 0 }}>Total Listings</StatLabel>
+        <Skeleton isLoaded={!contract || listingsQuery.isSuccess}>
+          <StatNumber>
+            {BigNumber.from(listingsQuery?.data || 0).toString()}
+          </StatNumber>
+        </Skeleton>
+      </Card>
+    </SimpleGrid>
+  );
+};
+
+interface ListingStatsV3Props {
+  contract?: MarketplaceV3;
+}
+
+export const ListingStatsV3: React.FC<ListingStatsV3Props> = ({ contract }) => {
```

Comment:
you need to detect here if the contract support these extensions

if direct listings, show count for direct listings
if english auctions, show count for english auctions
if both, show total listings count